### DISCUSSION TITLE
How to Guides

### DISCUSSION THREAD

**Discussion by deborahtrez** [curiouslearning/CuriousReaderInteractiveFunctionsH5P#223] [App Building Q&A]
Hello. I would love to create a Curious Reader book but I am not familiar with H5P.  Do you have any documentation of how-to guides that you can direct me to so that I can learn how to use the CR Authoring tool?

**Comment by lailahgrant** [comment]
Hi. I've really failed to start since I really don't know where to start from.

:cry: where do I go to add the login credentials that were emailed to us?
(Is there a link I can use to login e.g. curious-reader-contributor.com/login

**Reply by deborahtrez** [reply]
Hello @lailahgrant Im sorry that you're having trouble logging into the server. 
The link that you're looking for is [https://devcuriousreader.wpcomstaging.com/wp-admin](https://devcuriousreader.wpcomstaging.com/wp-admin) 
Once you get to the login, enter the username and password that have been provided to you in your email. Please let me know if you manage to login or if you're having any more trouble.
